frontend/src/components/Toast/Toast.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types'
+import { CheckIcon, CrossIcon } from '../Icon/icons'
+import BodyText from '../BodyText/BodyText'
+import Headline from '../Headline/Headline'
+
+function Toast({ className }) {
+  const toggleCloseBtn = () => {
+    const toastComponent = document.querySelector('.toast-wrapper')
+    toastComponent.classList.toggle('hide-toast-wrapper')
+  }
+
+  return (
+    <div
+      className={className}
+      style={{
+        background: '#402CAC',
+        maxWidth: '1212px',
+        maxHeight: '545px',
```

Comment:
What exactly the `maxWidth` and `maxHeight` would impact on the toast? And why these values?